com.unity.netcode.adapter.utp/CHANGELOG.md

```diff
@@ -5,25 +5,35 @@ All notable changes to this package will be documented in this file. The format
 
 ### Added
 
-- Added new 'Max Send Queue Size' configuration field in the inspector. This controls the size of the send queue that is used to accumulate small sends together and also acts as an overflow queue when there are too many in-flight packets or when other internal queues are full.
+### Changed
+
+### Fixed
+
+## [1.0.0-pre.4] - 2022-01-04
+
+### Added
+
+- Added new 'Max Send Queue Size' configuration field in the inspector. This controls the size of the send queue that is used to accumulate small sends together and also acts as an overflow queue when there are too many in-flight packets or when other internal queues are full. [#1491](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1491)
 
 ### Changed
 
-- Removed 'Maximum Packet Size' configuration field in the inspector. This would cause confusion since the maximum packet size is in effect always the MTU (1400 bytes on most platforms).
-- Updated com.unity.transport to 1.0.0-pre.10
-- All delivery methods now support fragmentation, meaning the 'Send Queue Batch Size' setting (which controls the maximum payload size) now applies to all delivery methods, not just reliable ones.
+- Updated Netcode for GameObjects dependency to 1.0.0-pre.4 [#1562](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1562)
+- Removed 'Maximum Packet Size' configuration field in the inspector. This would cause confusion since the maximum packet size is in effect always the MTU (1400 bytes on most platforms). [#1403](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1403)
+- Updated com.unity.transport to 1.0.0-pre.10 [#1501](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1501)
+- The 'Send Queue Batch Size' configuration field now controls the size of the send queue, rather than the size of a single batch of messages. Consequently, it should be set much higher than it was previously. [#1491](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1491)
```

Comment:
```suggestion
```

so the suggestion here is to remove this line like this right @simon-lemay-unity ?

Reply:
Yes, exactly!